ambry-messageformat/src/test/java/com.github.ambry.messageformat/MessageFormatRecordTest.java

```diff
@@ -41,18 +42,24 @@
   @Test
   public void deserializeTest() {
     try {
-      // Test delete V1 record
-      ByteBuffer deleteRecord = ByteBuffer.allocate(MessageFormatRecord.Delete_Format_V1.getDeleteRecordSize());
-      MessageFormatRecord.Delete_Format_V1.serializeDeleteRecord(deleteRecord, true);
-      deleteRecord.flip();
-      boolean deleted = MessageFormatRecord.deserializeDeleteRecord(new ByteBufferInputStream(deleteRecord));
-      Assert.assertEquals(deleted, true);
-
-      // corrupt delete V1 record
-      deleteRecord.flip();
-      deleteRecord.put(10, (byte) 4);
+      // Test Blob property V1 Record
```

Comment:
yep. This had merge conflict and I had to resolve it. Not sure why its showing up